tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml

```diff
@@ -85,10 +86,45 @@
 				<param name="regions" label="Study region or study subregions" optional="false" help="URL (stored in a .txt file) to the study region, or several regions, to classify the input data into groups of interest. Currently it has to be provided as a shapefile. It can be in any coordinate system and will be transformed to WGS84 during this process." type="data" format="txt"/>
 				<param name="input_data" label="Data table to be plotted on top of study regions" optional="false" help="URL (stored in a .txt file) to the input table containing the in-situ data points with coordinates." type="data" format="txt"/>
 				<param name="colname_long" label="Column name for longitude" optional="false" help="The name of the column containing longitude values for the data points." type="text"/>
-				<param name="colname_lat" label="Column name for latiitude" optional="false" help="The name of the column containing latitude values for the data points." type="text"/>
+				<param name="colname_lat" label="Column name for latitude" optional="false" help="The name of the column containing latitude values for the data points." type="text"/>
 				<param name="colname_value_name" label="Column name of data point identifier" optional="false" help="The name of the column containing identifier (e.g., site name) or values (e.g., depth) to color the points according to their corresponding values." type="text"/>
 				<param name="colname_region_id" label="Column name of region identifier" optional="false" help="The name of the column containing identifiers (e.g., basin name) to distinguish the polygons on the map if multiple regions are present." type="text"/>
 			</when>
+            <when value="owt-classification">
+				<param name="input_data_url" label="Input data" optional="false" help="URL (stored in a .txt file) to your input file." type="data" format="txt"/>
+				<param name="input_option" label="Type of input" optional="false" help="If type of input is csv, the output is a text file (.csv). If it is sat, the output is a NetCDF file (.nc) of the same dimensions and geo-reference as the input file." type="select">
+                    <option value="csv">csv: for text data input (first line wavelength, following lines remote-sensing reflectance)</option>
+                    <option value="sat">sat: for satellite product input containing reflectance (e.g., Sentinel-3 OLCI Level-2)</option>
+                </param>
+				<param name="sensor" label="Sensor name" optional="false" help="Spectral band configuration of satellite mission (includes adaptation to sensor spectral response functions)." type="select">
```

Comment:
the indentation is off here, how can this happen in an automatic generated script?

Reply:
Well, I need to admit that this update was not as automated as planned :). 
I always forget about the indentation. I think I need to revise my local linter...

Reply:
Use the force Luke. The Galaxy Language server!

Reply:
Oh, that is a good one!